__tests__/app.test.js

```diff
@@ -91,3 +91,27 @@ describe('GET /api/articles/:article_id', () => {
 			});
 	});
 });
+
+describe('GET /api/articles/', () => {
+	test('200: return all articles', () => {
+		return request(app)
+			.get(`/api/articles/`)
+			.expect(200)
+			.then(({ body }) => {
+				const { articles } = body;
+				console.log(typeof articles);
+				for (const article of articles) {
```

Comment:
we will need to check that the length of the array returned is the correct length as if this returns an empty array, the test will pass by default (as none of these expect statements will be executed)